src/test/skript/tests/syntaxes/effects/EffToggleCanPickUpItems.sk

```diff
@@ -0,0 +1,8 @@
+test "entity can pick up items":
+	spawn a zombie at spawn of "world"
+	set {_zombie} to last spawned zombie
+	allow {_zombie} to pick up items
+	assert {_zombie} can pick up items with "failed to allow zombie to pick up items"
+	forbid {_zombie} from picking items up
+	assert {_zombie} can't pick up items with "failed to disallow zombie to pick up items"
+	delete random entity of {_zombie}
```

Comment:
This could be a section. I also know that we can't add a drop item to this test due to the runtime of the tests. There is no delay as the zombie would pickup the item in 1-5 ticks after spawning, and not instantly. Something the JUnit test pull request will allow. So nothing can be done about that now.
```suggestion
	spawn a zombie at spawn of "world":
		allow event-entity to pick up items
		assert event-entity can pick up items with "failed to allow zombie to pick up items"
		forbid event-entity from picking items up
		assert event-entity can't pick up items with "failed to disallow zombie to pick up items"
		delete event-entity
```